components/brave_shields/resources/BUILD.gn

```diff
@@ -30,3 +30,24 @@ bundle_js("bundle_js_domain_block_interstitial") {
     "//ui/webui/resources/js:build_ts",
   ]
 }
+
+copy("resources") {
+  sources = [
+    "//brave/third_party/adblock-resources/src/filter_lists/list_catalog.json",
+    "//brave/third_party/adblock-resources/src/filter_lists/manifest.json",
+  ]
+
+  outputs =
+      [ "$root_out_dir/gkboaolpopklhgplhaaiboijnklogmbc/{{source_file_part}}" ]
```

Comment:
We use `gkboaolpopklhgplhaaiboijnklogmbc` multiple times in many .gn and  related files.
Looks like we need some gn contant for it to reduce copy-paste.